EngineErrorMap.md

```diff
@@ -3172,3 +3173,7 @@ Graph update has been interrupted since too many transitions(greater than %s) oc
 ### 14100
 
 Pool.destroy no longer take a function as parameter, Please specify destruct function in the construction of Pool instead
+
+### 14200
+
+Batcher2D.commitComp function had changed, please use commitPlainComp
```

Comment:
Batcher2D.commitComp could not be used for components without render data, please use commitPlainComp instead